beam-examples/README.md

```diff
@@ -0,0 +1,4 @@
+**Step 1**: Build Beam example maven project
+```bash
+mvn clean package
+```
```

Comment:
let's add a note here about where an example pipeline jar will land after we build in the example project